### PR TITLE
[nikon] Add Tokina ATX-i 11-20mm F2.8 CF lens

### PR DESCRIPTION
**Pull request by Sturmflut**
Test data from lensfun/lensfun#1255

### CODE REVIEW

**Comment by piponazo** [comment]
Hum ... weird, seems to be working on windows platforms but failing on Mac/Linux. Any idea why this might be happening?

File: tests/bugfixes/github/test_pr_2072.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+
+import system_tests
+
+class NikonSigmaLens_APO_MACRO_180_F35_EX_DG_HSM(metaclass=system_tests.CaseMeta):
+    url = "https://github.com/Exiv2/exiv2/pull/992"
+    
+    filename = "$data_path/Sigma_APO_MACRO_180_F3.5_EX_DG_HSM.exv"
```
Comment (piponazo) [inline tests/bugfixes/github/test_pr_2072.py:8]: There seems to be a copy/paste typo here. You are using a different file than the one added in this PR (also in the test name)

**Review by piponazo — CHANGES_REQUESTED** [review]

File: tests/bugfixes/github/test_pr_2072.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    commands = ["$exiv2 -pa --grep lensid/i $filename"]
+    stderr = [""]
+    stdout = [""
+        """Exif.NikonLd3.LensIDNumber                   Byte        1  Tokina ATX-i 11-20mm F2.8 CF
+"""
+]
```
Comment (hassec) [inline tests/bugfixes/github/test_pr_2072.py:14]: Could you try the below? 

```suggestion
    stdout = ["Exif.NikonLd3.LensIDNumber                   Byte        1  Tokina ATX-i 11-20mm F2.8 CF\n"]
```

**Comment by Sturmflut** [comment]
Fixed the test

**Review by piponazo — DISMISSED** [review]

**Review by piponazo — APPROVED** [review]

**Pull request merged**